CmdStarter.Tests.Commands/GlobalOptions/Filters/GlobalOptionFilterExcludeAll.cs

```diff
@@ -0,0 +1,21 @@
+﻿using com.cyberinternauts.csharp.CmdStarter.Tests.Common.Interfaces;
+
+namespace com.cyberinternauts.csharp.CmdStarter.Tests.Commands.GlobalOptions
```

Comment:
Isn't missing a last `.Filters` namespace?

This would apply to all three.

Reply:
Yes it is, fixed.